disco_aws_automation/disco_rds.py

```diff
@@ -87,12 +105,12 @@ def get_instance_parameters(self, instance_identifier):
             'DBSubnetGroupName': section,
             'Engine': db_engine,
             'EngineVersion': self.config_rds.get(section, 'engine_version'),
-            'Iops': int(self.config_with_default(section, 'iops')),
+            'Iops': self.config_integer(section, 'iops', 0),
```

Comment:
No provisioned IOPs, so either we'd have worse performance or we'd pay more (not sure which, I suspect the former).  We might want a higher starting value, but my assumption is that just not setting it is better than setting some arbitrary guess value in every DB configuration just to make it spin up. :-)